### DISCUSSION TITLE
failed while start docker and other services

### DISCUSSION THREAD

**Discussion by ysprt** [moby/moby#47425] [Q&A]
![image](https://github.com/moby/moby/assets/116148880/ed5956df-a89f-43fd-9a88-a7a51842d551)

please help me

**Comment by thaJeztah** [comment]
The docker service runs as a systemd unit, and systemd manages "dependency services" that must be available and running before the docker service can be started. Without more context it won't be possible to help you further, but you may be able to get more details from systemd using the `journalctl` command that is suggested (which may show what service isn't available or failed to start).
